src/Rule/Url.php

```diff
@@ -34,36 +34,59 @@ final class Url implements RuleWithOptionsInterface, SkipOnErrorInterface, WhenI
 
     public function __construct(
         /**
-         * @var string the regular expression used to validate the value.
+         * @var string The regular expression used to validate the value.
          * The pattern may contain a `{schemes}` token that will be replaced
          * by a regular expression which represents the {@see $schemes}.
          *
-         * Note that if you want to reuse the pattern in HTML5 input it should have ^ and $, should not have any
-         * modifiers and should not be case-insensitive.
+         * Note that if you want to reuse the pattern in HTML5 input, it should have `^` and `$`, should not have any
+         * modifiers, and should not be case-insensitive.
          */
         private string $pattern = '/^{schemes}:\/\/(([a-zA-Z0-9][a-zA-Z0-9_-]*)(\.[a-zA-Z0-9][a-zA-Z0-9_-]*)+)(?::\d{1,5})?([?\/#].*$|$)/',
         /**
-         * @var string[] list of URI schemes which should be considered valid. By default, http and https
+         * @var string[] List of URI schemes which should be considered valid. By default, http and https
          * are considered to be valid schemes.
          */
         private array $validSchemes = ['http', 'https'],
         /**
-         * @var bool whether validation process should take into account IDN (internationalized
-         * domain names). Defaults to false meaning that validation of URLs containing IDN will always
+         * @var bool Whether validation process should take into account IDN (internationalized
+         * domain names). Defaults to `false` meaning that validation of URLs containing IDN will always
```

Comment:
```suggestion
         * domain names). Defaults to `false` means that validation of URLs containing IDN will always
```

Reply:
This is actually wrong. in the other place it was "`false` meaning" -> "`false` means". This is OK because of "Defaults to" sentence start. I'll revert this.

Reply:
:heavy_check_mark: